src/webhooks/pubsub/slackNotifications.ts

```diff
@@ -152,12 +158,28 @@ export const constructSlackMessage = (
     const triageQueueIssues: SlackMessageUnorderedIssueItem[] = [];
     // Group issues into buckets based on time left until SLA
     let hasEnoughTimePassedSinceIssueCreation = false;
-    const addIssueToQueue = ({ url, number, title, triageBy, createdAt }) => {
+    const addIssueToQueue = ({
```

Comment:
Should we not modify [this](https://github.com/getsentry/eng-pipes/blob/f5d0058e4e449efa4e9cbaea4da9984510b03d28/src/webhooks/pubsub/slackNotifications.ts#L44) type definition to include `repo`, `org`, and `productArea` too?

Reply:
that is already done above, typescript caught that